App/src/main/java/com/google/sps/servlets/PopulateShelters.java

```diff
@@ -0,0 +1,35 @@
+// populates shelter list with dummy data
```

Comment:
Style thing: One thing my team does is to end every comment with a period. I used to think this was pointless and it took me a while to get into the habit, but it actually does have real value: It makes it very clear to the reader what is and is not a separate thought. For example, when reading these two adjacent comments, I at first thought it was just one - "populates shelter list with dummy data temporary, until ShelterMatch is completed."

Reply:
This is something I'll definitely be adopting. I better get in the habit of treating the comments as if I'm writing for someone else instead of myself. I believe those comments were from my previous code, those should also be deleted since the matching already works 👍